src/sources/http_scrape/integration_tests.rs

```diff
@@ -0,0 +1,229 @@
+//! Integration tests for http_scrape source.
+//! The container configuration file is `docker-compose-.http_scrape.yml`
+//! It leverages a static file server which serves the files in tests/data/http-scrape
+
+use tokio::time::{Duration, Instant};
+
+use crate::{
+    config::{ComponentKey, SourceConfig, SourceContext},
+    http::Auth,
+    serde::default_framing_message_based,
+    sources::http_scrape::scrape::NAME,
+    tls,
+    tls::TlsConfig,
+    SourceSender,
+};
+use codecs::decoding::DeserializerConfig;
+use vector_core::config::log_schema;
+
+use super::{
+    tests::{run_compliance, run_error, INTERVAL_SECS},
+    HttpScrapeConfig,
+};
+
+/// Logs (raw bytes) should be scraped and decoded successfully.
+#[tokio::test]
+async fn scraped_logs_bytes() {
+    let events = run_compliance(HttpScrapeConfig::new(
+        "http://dufs:5000/logs/bytes".to_string(),
```

Comment:
Somewhat of a nit, but we could improve the portability (not requiring `docker-compose`) by having helper functions that read env vars (`DUFS_ADDRESS`) set by the compose file otherwise falling back to a default of localhost or what not. That's used in (I think) all the other compose'd integration tests.

Reply:
Addressed in https://github.com/vectordotdev/vector/pull/13793/commits/5f7c44231f801eb5ee0be2f2a3549efb6c908e8c